### ISSUE TITLE
[FEATURE] It would be helpful to export the State for a sensor/scanner

### ISSUE THREAD

**Issue by edabe** [Benjamin-Stefan/ant-plus-next#34]
Hello Benjamin,
I found your library while looking for alternatives to ant-plus - thank you for creating this project!

**Is your feature request related to a problem? Please describe.**
This is not exactly a problem, it is more of an improvement: it would be helpful if the `state` of a sensor/scanner was exported.
I'm experimenting with your library and my callbacks need to define data as `any`, example:

```
hrScanner.on("heartRateData", (data: any) => {
    console.log(`id: ${data.DeviceId}`);
    console.dir(data);
});
```

However, if the state type was exported, my IDE could introspect into that making it a lot easier to use the library, and the code would be a bit cleaner as well.

**Describe the solution you'd like**
You might know better, but it seems to me that exporting the State as a type would do the trick

**Describe alternatives you've considered**
The alternative is to declare callbacks with the `data` payload defined as `any` 

**Additional context**
No additional context - I've just spent a few minutes looking around and thinking about how to port my current ant+ app to use your library

Thank you so much for your consideration!
Much appreciated.

**Comment by Benjamin-Stefan** [comment]
Hello edabe,

Thank you for your interest and feedback. I've looked into it, and the scanner and sensor types are now exported. I tested it in VSCode, and it works as expected.

I've also created a new example: examples/heartRateSensorTyped.ts.

Happy coding!

Best regards,
Benjamin

**Comment by edabe** [comment]
@Benjamin-Stefan : Thank you for considering this feature and for the quick turn around! I'll give it a try!

**Comment by edabe** [comment]
@Benjamin-Stefan : One thing I noticed: The state name for all scanners is `<profile>ScanState` (ex.: `FitnessEquipmentScanState`), except for heart rate: `HeartRateScannerState`.

Just a small inconsistency in the naming convention, I guess.

**Comment by Benjamin-Stefan** [comment]
Thank you for finding this. I have fixed that and in version 0.3.1 `HeartRateScannerState` is renamed to `HeartRateScanState`.
